### PR TITLE
Adds MOCHA and LERC

### PR DESCRIPTION
**Pull request by dirkgr**
@anthonywchen, what do you think?

### CODE REVIEW

**Comment by anthonywchen** [comment]
This looks great, thank you!

File: app/app/projects/vilbert_vqa/description.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+This work implements the VQA task on VilBERT. It does not reproduce the pre-training regime
```
Comment (epwalsh) [inline app/app/projects/vilbert_vqa/description.md:1]:
```suggestion
This work implements the VQA task with ViLBERT. It does not reproduce the pre-training regime
```

File: app/app/projects/vilbert_vqa/description.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+that the original paper goes through, and as such, does not reproduce the scores from the paper.
+
+It achieves 41% F1 and 52% VQA. If you want to produce SOTA scores with this model, please
```
Comment (epwalsh) [inline app/app/projects/vilbert_vqa/description.md:4]:
```suggestion
It achieves 41% F1 and 52% VQA score. If you want to produce SOTA scores with this model, please
```

**Review by epwalsh — APPROVED** [review]
LGTM! Just a couple of wording suggestions.

**Pull request merged**